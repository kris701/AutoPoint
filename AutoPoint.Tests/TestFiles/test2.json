{
  "Includes": [],
  "Branch": {
    "name": "simple2",
    "Nodes": [
      {
        "name": "test",
        "route": "abc"
      }
    ]
  }
}
