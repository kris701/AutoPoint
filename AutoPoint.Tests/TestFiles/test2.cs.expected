// This document is auto generated!
public static class simple2 {
	public const string Name = "simple2";
	public const string test = "simple2/abc";
}
